audit_engine/dynamic_analysis/__init__.py

```
@@ -66,31 +80,139 @@ class DynamicAnalysisOrchestrator:
+            echidna_config = self.config.get("echidna", {})
+            adversarial_config = self.config.get("adversarial_fuzz", {})
+        
+        # Standard Echidna fuzzing
+        if enable_echidna:
+            try:
                 adapters.append(EchidnaAdapter(config=echidna_config, logger=self.logger))
-            
-            # Adversarial fuzzing as per Rahman et al. (2025)
-            if self.config.get("enable_adversarial_fuzz", True):
-                adversarial_config = self.config.get("adversarial_fuzz", {})
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)